music21/abcFormat/translate.py

```diff
@@ -1031,20 +1031,6 @@ def testCleanFlat(self):
         self.assertEqual(cs.bass(), pitch.Pitch('B-2'))
         self.assertEqual(cs.pitches[0], pitch.Pitch('B-2'))
 
-        # common.cleanedFlatNotation() shouldn't be called by
```

Comment:
Can you explain why these tests are removed?

Reply:
These tests where testing the fact that the root with 'bb3' should not be interpreted as b-3. Since I fixed some bugs with chord alterations, this interfered with the new code, so I lazily removed them.
Shouldn't have, so I put them back and corrected it: now b3 is interpreted as a chord alteration, and D# becomes D. Also, the pitches are one octave lower because of the '_hasPitchAboveC4' test.